package.json

```diff
@@ -50,8 +50,6 @@
     "terser": "^5.37.0",
     "typescript": "^5.7.3",
     "vite": "^6.0.7",
-    "vite-bundle-visualizer": "^1.2.1",
```

Comment:
This visualizer is used - `build:analyze` - shows you which packages are included and their size.

Reply:
Added it back.
Although it probably should be used though [pnpm dlx](https://pnpm.io/cli/dlx).